codeforamerica/vita-min
# NJ 47 - add w2 wages sum to PDF and XML

## Link to pivotal/JIRA issue
- https://github.com/newjersey/affordability-pm/issues/47

## Is PM acceptance required? (delete one)
- Yes - don't merge until issue is accepted!

## What was done?
- Sum `StateWagesAmt` from all W2s from federal DF, and round to nearest whole number
- Output sum in NJ XML field `WagesSalariesTips`
- Output sum in Line 15 of PDF
- [code] update zeus_two_w2s and zeus_many_w2s to use more interesting wage numbers with decimals

## How to test?
- Use `zeus_minimal` for a user with no DF W2s to confirm XML are not filled in when no W2
- Use `zeus_two_w2s` to test with wages: `12345.67` + `50000`
- Use `zeus_many_w2s` to test with wages: `50000.33` + `50000.33` + `50000.33` + `50000.33`
  
## Screenshots (for visual changes)
![image](https://github.com/user-attachments/assets/7e1b977d-9457-4eac-bbb1-806b92a2ceaa)
![image](https://github.com/user-attachments/assets/3a5bcf01-f728-4f3b-a115-3b6ab5c9a82f)
